include/exadg/aero_acoustic/single_field_solvers/acoustics.h

```diff
@@ -68,6 +68,43 @@ class SolverAcoustic
     time_integrator->setup(application->get_parameters().restarted_simulation);
   }
 
+  void
+  advance_multiple_timesteps(double const global_dt)
+  {
+    double const dt = compute_sub_time_step_size(global_dt);
+
+    // define epsilon dependent on time-step size to avoid
+    // wrong terminations of the sub-timestepping loop.
+    double eps = 0.01 * dt;
+
+    // perform time-steps until we advanced the global time step
+    unsigned int n_sub_time_steps = 0;
+    while(n_sub_time_steps * dt + eps < global_dt)
+    {
+      time_integrator->set_current_time_step_size(dt);
+      time_integrator->advance_one_timestep();
+      ++n_sub_time_steps;
+    }
+
+    if(time_integrator->started())
+    {
+      ++sub_time_steps.first;
+      sub_time_steps.second += n_sub_time_steps;
+    }
+  }
+
+  double
+  get_average_sub_time_steps() const
```

Comment:
please extend name to  `_average_number_of_`

Reply:
[a132791](https://github.com/exadg/exadg/pull/645/commits/a132791a4dc04a4b4bfd4286c0c6f9c2638ed074)